source/features/show-user-top-repositories.tsx

```diff
@@ -12,8 +12,12 @@ function init(): void {
 		sort: 'stargazers',
 	}).toString();
 
-	// Showcase title
-	select('.js-pinned-items-reorder-container .text-normal')!.firstChild!.after(
+	// If already added, skip
+	const pinnedText = select('.js-pinned-items-reorder-container .text-normal')!;
+	if(pinnedText.innerHTML.includes("Top repositories")) return;
```

Comment:
This was the cleanest solution I could come up with, inner text wasn't working out since the elements added after the <p> tag but I can tweak it if theres a more comfortable workaround here. I figured it'd either be parsing the innerHTML or adding a data attribute and making another select to the page which would be cleaner but very very very slightly less performant lol